ubuntu/22.04/install.sh

```diff
@@ -655,10 +655,13 @@ EOF
 	fi
 
 	if [ "X$ENABLE_SFC_HBN" == "Xyes" ]; then
-		NUM_VFs_PHYS_PORT0=${NUM_VFs_PHYS_PORT0:-14}
-		NUM_VFs_PHYS_PORT1=${NUM_VFs_PHYS_PORT1:-0}
+		HBN_UPLINKS=${HBN_UPLINKS:-"p0,p1"}
+		HBN_REPS=${HBN_REPS:-"pf0hpf,pf1hpf,pf0vf0-pf0vf13"}
+		HBN_DPU_SFS=${HBN_DPU_SFS:-"pf0dpu1,pf0dpu3"}
+		HUGEPAGE_SIZE=${HUGEPAGE_SIZE:-2048}
+		HUGEPAGE_COUNT=${HUGEPAGE_COUNT:-4096}
 		log "INFO: Installing SFC HBN environment"
-		chroot /mnt /opt/mellanox/sfc-hbn/install.sh --ecpf0 $NUM_VFs_PHYS_PORT0 --ecpf1 $NUM_VFs_PHYS_PORT1
```

Comment:
Need to ensure we keep backword compatibility with old version for atleast one release.

Reply:
oops, I thought we wanted to switch to the new input.

I folded in additional checks